docker-for-mac/osxfs-caching.md

```diff
@@ -0,0 +1,195 @@
+---
+description: OSXFS CACHING
+keywords: mac, osxfs, volume mounts, docker run -v, performance
+title: Performance tuning for volume mounts (shared filesystems)
+toc_max: 4
+toc_min: 2
+---
+
+[Docker 17.04 CE Edge](https://docs.docker.com/edge/#docker-ce-edge-new-features) adds support for two new flags to the [docker run -v, volume](https://docs.docker.com/engine/reference/run/#volume-shared-filesystems)
+option, `cached` and `delegated`, that can significantly improve the
+performance of mounted volume access on Docker for Mac. These options begin to solve some of the challenges discussed in [Performance issues, solutions, and roadmap](/docker-for-mac/osxfs.md#performance-issues-solutions-and-roadmap).
+
+> **Tip:** Release notes for Docker CE Edge 17.04 are [here](https://github.com/moby/moby/releases/tag/v17.04.0-ce), and the associated pull request for the additional `docker run -v` flags is [here](https://github.com/moby/moby/pull/31047).
+
+## Performance implications of host-container file system consistency
+
+With Docker distributions for an increasing number of platforms,
```

Comment:
Perhaps reads better if "With Docker distributions now available for an <...>" (missing "available")